src/bin/lttng-sessiond/Makefile.am

```diff
@@ -88,5 +85,13 @@ lttng_sessiond_LDADD = -lurcu-common $(URCU_LIBS) $(KMOD_LIBS) \
 
 
 if HAVE_LIBLTTNG_UST_CTL
-lttng_sessiond_LDADD += $(UST_CTL_LIBS)
+liblttng_sessiond_comm_la_LIBADD += $(UST_CTL_LIBS)
 endif
+
+bin_PROGRAMS = lttng-sessiond
+
+lttng_sessiond_SOURCES = lttng-sessiond.h main.c
+
+lttng_sessiond_LDFLAGS = -rdynamic
```

Comment:
Can you detail why you add -rdynamic here?

Reply:
It was not added, I simply moved the `lttng_sessiond_LDFLAGS = -rdynamic` line for organizational reasons.

Reply:
Sorry, I missed that, all good then.